get_covid_clade_counts.py

```diff
@@ -0,0 +1,81 @@
+"""
+Create a parquet file with clade counts by date and location and save it to an S3 bucket.
+
+This script wraps the virus-clade-utils package, which generates the clade counts using the
+GenBank-based Sars-CoV-2 sequence metadata from Nextstrain.
+https://github.com/reichlab/virus-clade-utils
+
+The script is scheduled to run every Monday, for use in the modeling round that will open
+on the following Wednesday.
+
+To run the script manually:
+1. Install uv on your machine: https://docs.astral.sh/uv/getting-started/installation/
+2. From the root of this repo: uv run get_covid_clade_counts.py --as-of=YYYY-MM-DD
+"""
+
+# /// script
+# requires-python = ">=3.11"
+# dependencies = [
+#   "click",
+#   "virus_clade_utils@git+https://github.com/reichlab/virus-clade-utils"
+# ]
+# ///
+
+import click
+import os
+import logging
+
+from virus_clade_utils.cladetime import CladeTime  # type: ignore
+from virus_clade_utils.util.sequence import filter_covid_genome_metadata, get_clade_counts  # type: ignore
+
+# Log to stdout
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler()
+formatter = logging.Formatter(
+    "%(asctime)s -  %(levelname)s - %(message)s", datefmt="%m/%d/%Y %I:%M:%S %p"
+)
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
+
+
+data_dir = "./data"
```

Comment:
If you use `pathlib` to do this, it should ["do the right thing" for people working on different systems](https://docs.python.org/3/library/pathlib.html#pathlib.Path) (Windows, for example). And I learned recently that `Path` has a `mkdir` method, so you can do it all without `os` 🎉 

```
from pathlib import Path

data_dir = Path("./data")
data_dir.mkdir(exist_ok=True)
```

Reply:
done